fiboa_cli/datasets/us_ca_scm.py

```diff
@@ -0,0 +1,77 @@
+from os.path import join, dirname
+
+from .commons.ec import load_ec_mapping
+from ..convert_utils import convert as convert_
+
+
+SOURCES = {
+    "https://data.cnra.ca.gov/dataset/6c3d65e3-35bb-49e1-a51e-49d5a2cf09a9/resource/f38d3f6f-dcf1-4553-9f07-4f381d494320/download/i15_crop_mapping_2022_provisional_gdb.zip": ["i15_Crop_Mapping_2022_Provisional_GDB/i15_Crop_Mapping_2022_Provisional.gdb"]
+}
+ID = "us_ca_scm"
+SHORT_NAME = "CA SCM"
```

Comment:
```suggestion
SHORT_NAME = "US, California (SCM)"
```

That would be a bit more consistent with other "state-level" converters.